manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -40,20 +42,22 @@ meta:
     release: datadog-for-cloudfoundry
   - name: cloud_controller_worker
     release: capi
+  - name: metron_agent
+    release: (( grab meta.release.name ))
 
   clock_templates:
-  - name: consul_agent
-    release: (( grab meta.consul_templates.consul_agent.release ))
```

Comment:
I think this needs to remain now that we've upgraded to CF 245.  Part of the changes are that several cf components now use consul to locate the etcd cluster.  See https://github.com/cloudfoundry/cf-release/commit/c432b73#diff-b9982126ee4438e20576a100670694d8R1275 for the upstream manifest change.